content/applications/hr/recruitment.rst

```diff
@@ -1,10 +1,730 @@
+  formatted text, such as numbered lists, headings, tables, as well as links, photos, and more. Type
+  `/` and a list of options are presented. Scroll through the options and click on the desired item.
+  The item appears in the field and can be modified. Each command presents a different pop-up
+  window. Follow the instructions for each command to complete the entry.
+- :guilabel:`Reminders`: Select a reminder from the drop-down menu. Options include
+  :guilabel:`Notification`, :guilabel:`Email`, and :guilabel:`SMS Text Message`, each with a
+  specific time period before the event (hours, days, etc). The reminder chosen will alert the
+  meeting participants of the meeting via the selected option.
+- :guilabel:`Location`: Enter the location for the meeting.
+- :guilabel:`Meeting URL`: If the meeting is virtual, enter the meeting link in this field.
+- :guilabel:`Tags`: Select any tag(s) for the meeting using the drop-down menu. There is no limit to
+  the number of tags that can be used.
+
+Options tab
+^^^^^^^^^^^
+
+- :guilabel:`Recurrent`: If the meeting should repeat at a selected interval (not typical for a
+  first interview), check the box next to :guilabel:`Recurrent`. This reveals a recurrence module.
+  Enter the details for how often the meeting should repeat, including an end time.
+- :guilabel:`Privacy`: Select the visibility of this meeting. Options are :guilabel:`Public`,
+  :guilabel:`Private`, and :guilabel:`Only internal users`. :guilabel:`Public` allows for everyone
+  to see the meeting, :guilabel:`Private` allows only the attendees listed on the meeting to see the
+  meeting, and :guilabel:`Only internal users` allows everyone in the company to see the meeting.
+- :guilabel:`Show as`: Select from the drop-down whether the meeting time should appear as
+  :guilabel:`Busy` or :guilabel:`Available` in the attendees' schedules. Setting this field to
+  :guilabel:`Available` allows other meetings to be scheduled for the same time. It is recommended
+  to set this field to :guilabel:`Busy`, so no other meetings can be scheduled for the attendees at
+  the same time.
+
+.. image:: recruitment/new-event.png
+   :align: center
+   :alt: A new meeting card with all the details populated, and ready to save.
+
+Send meeting to attendees
+*************************
+
+Once changes have been entered and the meeting details are correct, the meeting can be sent to the
+attendees via email or text message.
+
+To send the meeting via email, click the :guilabel:`Email` button next to the list of attendees. A
+:guilabel:`Contact Attendees` pop-up appears. A pre-formatted email using the default
+:guilabel:`Calendar: Event Update` email template populates the email body field. The followers of
+the document (job application), as well as the :guilabel:`Recruiter` are added as
+:guilabel:`Recipients` by default. Add the applicant's email address to the list to send the email
```

Comment:
The organzer is not added on as a attendee and it doesn't add them onto their schedule. 
```suggestion
:guilabel:`Calendar: Event Update` email template populates the email body field. The followers of
the document (job application), as well as the :guilabel:`Recruiter` are added as
:guilabel:`Recipients` by default. Add the applicant's email address to the list to send the email
```
The above statement isn't true

Reply:
I just tested this and rechecked everything a couple times. I don't mention the organizer, so I don't know what part of this you are referring to. I did retry this and you are correct- I was logged in as the recruiter, so it confused me, I thought it was adding the recruiter- it's just the followers and whoever is making the meeting,